arm-machinelearning/2016-05-01-preview/swagger/webservices.json

```diff
@@ -322,33 +351,32 @@
       "properties": {
         "title": {
           "type": "string",
-          "description": "The title of the Azure ML web service."
+          "description": "The title of the web service."
         },
         "description": {
           "type": "string",
-          "description": "The description of the Azure ML web service."
+          "description": "The description of the web service."
         },
         "createdOn": {
           "type": "string",
-          "description": "The moment of time the Azure ML web service was created.",
+          "description": "Read Only: The date and time when the web service was created.",
```

Comment:
@BryanLa - I do not like this aspect of documentation. Why would you want the spec author to duplicate information in the description? The documentation tool should pickup `"readOnly": true` and update the documentation accordingly.